src/lib/cimb_translator/test/CimbReaderTest.cpp

```diff
@@ -20,6 +19,7 @@ namespace {
 		return s;
 	}
 }
+#include "serialize/str_join.h"
```

Comment:
The `operator<<` in question is defined in the anonymous namespace right above this line.